### PR TITLE
ADAPT-2728: Replacing intro text with emptySearchText if input empty

### PR DESCRIPTION
**Pull request by Lyafel**
# Summary
- Added a helper text for when the search input in submitted empty.

# Review By (Date)
- When does this need to be reviewed by?

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Review Tasks

## Setup tasks and/or behavior to test

1. Navigate to https://deploy-preview-91--stanford-alumni.netlify.app/search/
2. Check out the functionality on the search modal

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

**Review by sherakama — CHANGES_REQUESTED** [review]
@Lyafel,

I am not seeing the empty search text show up on the modal and clicking on the search button is still kicking the user over to the search page when empty.

https://share.getcloudapp.com/d5uAZz1E

**Comment by Lyafel** [comment]
Fixed the redirect regression after merge. Trying to figure out why doesn't the message appear

**Comment by Lyafel** [comment]
Ready for review

File: src/components/search/searchModal.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import React from "react";
+import React, { useState } from "react";
 import { Heading } from "decanter-react";
-import { useStaticQuery, graphql } from "gatsby";
+import {useStaticQuery, graphql, navigate} from "gatsby";
```
Comment (mattanglin) [inline src/components/search/searchModal.js:3]: Looks like this line is causing lint errors (prettier requires spaces between brackets). Should be fixable by running `> npm run lint:fix` locally.

**Review by mattanglin — CHANGES_REQUESTED** [review]
Aside from the lint error, lgtm! I noticed another bug while testing, but it seemed edge case-y and unrelated to this PR so I [created another ticket for it](https://stanfordits.atlassian.net/browse/ADAPT-3149).

- [ ] Fix lint error

**Review by sherakama — APPROVED** [review]
GTG when linting is resolved. @Lyafel

**Review by mattanglin — APPROVED** [review]
👍🏼
